tigerpath/majors_and_certificates/majors/COM_2018.json

```
@@ -0,0 +1,263 @@
+          "course_list": [
+            "LANG 2*",
+            "LANG 3*",
+            "LANG 4*",
+            "LANG 5*"
+          ]
+        },
+        {
+          "name": "Tracks",
+          "max_counted": 1,
+          "min_needed": 1,
+          "explanation": "The remaining two of the nine required courses are taken in appropriate departments or programs throughout the university. Course selections generally fall into one of five tracks listed below. Each represents the study of literature in a different comparative context.",
+          "req_list": [
+            {
+              "name": "Comparative Work in Literatures",
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "<i>Editor's Note: Most of these courses fall under this requirement but as TigerPath does not verify this.</i>\n\nComparative work in literatures: two 300-, 400-, or 500-level courses in any literature, including English. These courses need not be taken in the same department; a pairing such as ENG 320 and SPA 354, for example, would fulfill the requirement.",

```

Text typo
Remove the "as"
```suggestion
              "explanation": "<i>Editor's Note: Most of these courses fall under this requirement but TigerPath does not verify this.</i>\n\nComparative work in literatures: two 300-, 400-, or 500-level courses in any literature, including English. These courses need not be taken in the same department; a pairing such as ENG 320 and SPA 354, for example, would fulfill the requirement.",

```